### PR TITLE
[IMP] mail: remove alias mixin from channel

### PR DESCRIPTION
**Pull request by rapr-odoo**
**Current behavior before PR:**

`mail.alias.mixin` is being used in `mail.channel`

**Desired behavior after PR is merged:**

removed `mail.alias.mixin` with all of its uses in `mail.channel`

Task-2823691

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

**Review by seb-odoo — APPROVED** [review]
Hello @tde-banana-odoo I believe this is a remaining from mail group split, looks ok for you to remove it from channel?

**Comment by tde-banana-odoo** [comment]
Hello,

I don't have any strong opinion. Just pay attention that people may have aliases, normally this should still work but contact-based aliases (normally based on members) may be broken as some override may not work anymore (unless it has been removed, did not check).

**Review by seb-odoo — APPROVED** [review]
@robodoo r+

Still waiting on upgrade approval

**Comment by seb-odoo** [comment]
@robodoo r+

**Pull request closed without merging**
